### ISSUE TITLE
Button status

### ISSUE THREAD

**Issue by zcjess** [zcjess/Spire#7]
The expandable buttons should have aria-expanded w/o aria-haspopup as attributes.
![image](https://github.com/zcjess/Spire/assets/90011962/43935a0b-eb45-4a1a-a568-98a084e90306)
![image](https://github.com/zcjess/Spire/assets/90011962/600fd695-9214-4171-a7a7-bfad8062ee4c)
![image](https://github.com/zcjess/Spire/assets/90011962/24600de7-385b-4c98-bcae-240ace6d2fdf)

<img width="761" alt="image" src="https://github.com/zcjess/Spire/assets/90011962/b3bfa646-7d1b-4810-bf68-6fc9916f596f">

<img width="662" alt="image" src="https://github.com/zcjess/Spire/assets/90011962/64b9eeda-a70c-4ad7-a01d-f98b18b69a43">
<img width="643" alt="image" src="https://github.com/zcjess/Spire/assets/90011962/788afa94-90c8-4711-bbc5-302fa22a1251">

<img width="652" alt="image" src="https://github.com/zcjess/Spire/assets/90011962/0196dbba-4a2a-484c-920b-01930b0485a1">

**Comment by zcjess** [comment]
announce pressed buttons, aria-pressed, to let users know which page they are currently on.
<img width="379" alt="image" src="https://github.com/zcjess/Spire/assets/90011962/36c6c106-6520-4bc7-93ef-9eb4a127219d">
<img width="227" alt="image" src="https://github.com/zcjess/Spire/assets/90011962/7a25b00e-c46c-44c9-ad90-2a7af9336446">

**Comment by zcjess** [comment]
Better have a sound feedback after users successfully select and add a tag
<img width="932" alt="image" src="https://github.com/zcjess/Spire/assets/90011962/e512bff0-332a-4949-977e-21439c0ec3e5">
